releasenotes/notes/add-updateinterval-args.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+  - |
+    **Added** a flag setting `updateInterval` that is the interval that status reporter updates status and its default value is 500ms.
```

Based on the above comment: 

```suggestion
    **Added** a flag, `updateInterval`, to configure the interval to update the XDS distribution status at. This defaults to 500ms.
```